spec/support/question_set_helper.rb

```diff
@@ -1,5 +1,11 @@
 module QuestionSetHelper
-  VALID_ANSWERS = ['309 Colver Rd', '812', 'Jasper', '49230', 'None Of The Above']
+  VALID_ANSWERS = [
+    '309 Colver Rd',
+    '812',
+    'Jasper',
+    '49230',
+    'None Of The Above'
+  ]
```

Comment:
In my code I usually would apply IceNine, but it is not part of this API gem. SO I will freeze the strings and the array.